api/src/main/resources/workbench-api.yaml

```diff
@@ -5932,6 +5932,8 @@ definitions:
 
   AccessModuleStatus:
     type: object
+    required:
+      - moduleName
```

Comment:
this is often helpful when we have only this status object and no other context.  Now it's mandatory.